postgres-dockerfile/Dockerfile

```diff
@@ -1,13 +1,29 @@
 FROM postgres:9.5
 MAINTAINER jeffsturgis@gmail.com
 
+ARG DEBIAN_FRONTEND="noninteractive"
```

Comment:
Seems the docker version on CI does not recognise `ARG`, maybe you could use `ENV` as a patch? https://travis-ci.org/jsturgis/musicbrainz-docker/builds/138642665